docs/core/integration-quality-scale/rules/iqs040.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IQS040 - The documentation includes a high-level description of the brand, product, or service it integrates with; if available, with links to the brand, product, or service website"
```

Comment:
Do we need to include all the requirements in the rule name? I'd shorten it and specify the exact requirements below in the description of the rule.